src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,13 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  if (number === 1) {
+    return [number, 15, 15];
+  }
+  if (number === 2) {
+    return [number, 24, 24];
+  }
+  if (number >= 3) {
+    return [number, 24 + (number - 2) * 4, 24 + (number - 2) * 5];
+  }
+};
```

Comment:
We have the same eslint error caused by not having a default exit condition here. As an example, what would happen if 0 was passed in? You have a working condition for any number greater than or equal to 2 here, so I would add a condition for zero at the start in addition to your others, then make the greater than 2 function your default:
```
const humanCatDogYears = (number) => {
  if (number === 0) {
    return [0, 0, 0];
  }
  if (number === 1) {
    return [number, 15, 15];
  }
  if (number === 2) {
    return [number, 24, 24];
  }
  return [number, 24 + (number - 2) * 4, 24 + (number - 2) * 5];
};
```